libs/core/langchain_core/utils/image.py

```diff
@@ -1,29 +1,8 @@
-import base64
-import mimetypes
+from typing import Any
 
 
-def encode_image(image_path: str) -> str:
-    """Get base64 string from image URI.
-
-    Args:
-        image_path: The path to the image.
-
-    Returns:
-        The base64 string of the image.
-    """
-    with open(image_path, "rb") as image_file:
-        return base64.b64encode(image_file.read()).decode("utf-8")
-
-
-def image_to_data_url(image_path: str) -> str:
-    """Get data URL from image URI.
-
-    Args:
-        image_path: The path to the image.
-
-    Returns:
-        The data URL of the image.
-    """
-    encoding = encode_image(image_path)
-    mime_type = mimetypes.guess_type(image_path)[0]
-    return f"data:{mime_type};base64,{encoding}"
+def __getattr__(name: str) -> Any:
+    if name in ("encode_image", "image_to_data_url"):
+        msg = f"'{name}' has been removed for security reasons."
```

Comment:
This is very cryptic!

1) why did we remove encode_image / image_to_data_url are those not fine if users just use them as a utility? (i also get that they're really light weight so maybe we don't need them.)
2) The error message is cryptic. Could we provide at least one more detail? It's not clear why the utility functions themselves would be a security concern.


The main concern is that this going to break user code, so would be ideal to provide information. Including the version as of which this change was made